VUL-262 Support multiple ABI

ramilexe opened the pull request.

ramilexe committed:
VUL-262 Support multiple ABI

reenko commented on src/utils/index.ts:
```diff
@@ -0,0 +1,5 @@
+import Contract from "../models/contract/contract";
+
+export const getContractsFromLogs = (contracts: Contract[], logs: string[]): Contract[] => {
```
Contract[] | ContractFunction

ramilexe committed:
remove unused code

ramilexe committed:
fix db migration, new unit tests

ramilexe committed:
fix linter error

ramilexe committed:
fix linter
type for ABI, graphql entities (EthReceiptCid, EthTransactionCid, EthHeaderCid)
fixed bug with saving duplicated header_cid and transaction_cid
fixed bug with saving event

ramilexe committed:
rename graphql quires
